.github/workflows/pr-checks.yml

```diff
@@ -1,92 +0,0 @@
-name: PR checks
-
-on: 
-  pull_request:
-    branches: [ master ]
-
-env:
```

Comment:
are these not needed anymore if we split up those checks?

Reply:
Yes, we don't need this file.